xml/x509_issuers.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+	<fingerprints matches="x509.issuer" protocol="x509">
+    <!--
+  This fingerprint set matches the Issuer field of x509 certificates. These x509
+  certificates may be sourced from any SSL or TLS service. If a particular system
+  has identical subject and issuer fields, the subject field should be preferred.
+  -->
+    <!-- Chromecast and various devices that support the Cast protocol -->
+    
+	<fingerprint pattern="^CN=Chromecast ICA (\d+)\s*\(?([^,\)]*)\)?,OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US$">
+        <description>Google Chromecast</description>
+        <example chromecast.generation="3">CN=Chromecast ICA 3,OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <example chromecast.generation="4" chromecast.capabilities="Audio">CN=Chromecast ICA 4 (Audio),OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <example chromecast.generation="5" chromecast.capabilities="4K">CN=Chromecast ICA 5 (4K),OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <example chromecast.generation="6" chromecast.capabilities="Audio Assist">CN=Chromecast ICA 6 (Audio Assist),OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <example chromecast.generation="7" chromecast.capabilities="Audio Assist 2">CN=Chromecast ICA 7 (Audio Assist 2),OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <example chromecast.generation="9" chromecast.capabilities="Audio Assist">CN=Chromecast ICA 9 (Audio Assist),OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <example chromecast.generation="11" chromecast.capabilities="Video Assist">CN=Chromecast ICA 11 (Video Assist),OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <example chromecast.generation="12">CN=Chromecast ICA 12,OU=Cast,O=Google Inc,L=Mountain View,ST=California,C=US</example>
+        <param pos="0" name="os.vendor" value="Google" />
+        <param pos="0" name="os.product" value="ChromeOS" />
+        <param pos="0" name="hw.device" value="Media Server" />
+        <param pos="0" name="hw.vendor" value="Google" />
+        <param pos="0" name="hw.product" value="Chromecast" />
+        <param pos="0" name="hw.cpe23" value="cpe:/a:google:chromecast:-" />
+        <param pos="1" name="chromecast.generation" />
+        <param pos="2" name="chromecast.capabilities" />
+    </fingerprint>
+    
+	<fingerprint pattern="^CN=Asus fugu Cast ICA,OU=Widevine,O=Google Inc,L=Kirkland,ST=Washington,C=US$">
```

Comment:
Can you comment on how you are generating these DN strings?  I thought they were going to be ordered alphabetically by field name but that does not appear to be the case.  If there is some other order we should standardize around then we should discuss it.

Reply:
These are the Issuer and Subject functions of https://golang.org/pkg/crypto/x509/#Certificate

Reply:
The problem is that [RFC 2253](http://www6.ietf.org/rfc/rfc2253.txt) doesn't define an order:

>    When converting from an ASN.1 RelativeDistinguishedName to a string,
   the output consists of the string encodings of each
   AttributeTypeAndValue (according to 2.3), in any order.

IMO we should either agree upon a specific order and list the field names and their order in these XML files as a comment or require that the field names be ordered alphabetically.  

I agree that the format used by Go is the most common and will probably work for many other languages and would likely result in little/no conversion in most cases, however.

Reply:
Thanks! I added comments for now.